Documentation/Signoffs/PowerSystemV2.md

```diff
@@ -0,0 +1,93 @@
+
+###### 3. Diode Selection 
+Due to the uncertainty of not knowing the components in the OEM components, an assumption is made that energy storage elements are implemented somewhere. The switch system needs to be designed to allow for the load to always have a path for current to flow and dissipate the stored energy. A flyback diode is introduced to be in parallel with the load and allow energy to circulate through the load until it dissipates completely after the switch is opened.
+The diode for the circuit shown in Figure 1 will be a 19TQ015CJ from SMC Diode Solutions [2]. It is designed to have a continuous forward voltage of 360 mV at a maximum continuous current of 19 A. The datasheet provides that the diode can handle a peak current of 120 A for an instance before dissipating, which is three times greater than the maximum current that can be supplied.
+
+###### 4. Wiring
+The wiring to all of the external components will be 18-gauge wire. This is based around the pre-terminated wire that is sold to be used for the Arduino’s DC jack [3]. The maximum current that is allowed for 18-gauge wire is 2.3 Amps [4]. The current drawn by each external microcontroller or sensor is expected to have a maximum of 70 mA per their respective datasheet. This 70 mA current is within the 2.3 A limit of the 18-gauge wire. The internal resistance of 18-gauge wire is approximately 6.385 Ohms per 1000 ft. For the 6 ft of wire, the resistance is 0.03831 Ohms. This means that the maximum voltage across the wire is 0.08811 V or 88.11 mV.  
+
+###### 5. Voltages and Fuses
+Per the datasheet for the Arduino Uno, the microcontroller draws no more than 70 mA and can receive voltages ranging from 6 to 20 volts on the DC jack [5]. The microcontrollers will then be connected directly to the 12 V system due to 12 +/- 0.4 V falling within the range. The rotary encoder used for the steering system states that it can draw up to 70 mA per the data sheet at a range of 10.6 to 24 V. Likewise, the voltage will be within the requirements with the 12 V supply.
+A fuse panel will be implemented to protect the wiring from short circuits. The fuses for each system will be 100 mA glass fuses from Digi-Key [6]. These fuses will be used in series as the first point of failure instead of the wire. It is common practice for mechanics and electricians to install wiring for devices that would be around 60-85% of the maximum current rating for the expected load. The 100 mA fuse was chosen to allow for approximately 70% of the maximum tolerance to allow for the event of a device drawing slightly more current than expected. 
+To protect the wiring between the power supply and the switch, the internal fuse for the power supply will be the protection. This is designed within the supply and eliminates any issues seen by the terminals.
+
+![image](https://user-images.githubusercontent.com/117474411/217126300-f78afd84-9ca5-42df-b907-6aa4385e1d3c.png)
```

Comment:
So, the devices are intended to pull 70ma and you are fusing at that value? Typically, that is not done because the fuse could blow in normal use. Overcurrent protection is typically only used to protect the conductors in which case you should have the fuse be $WireAmpacitiy \geq Fuse \geq 1.25 \cdot I_c$ where I is the normal current in the conductor (70ma).

Reply:
And this equation is not satisfied with the previously mentioned 100 mA fuse and 18 gauge wire rated at 2.3 A?

Reply:
I can't tell based on the design as the values aren't labeled. But the values in the simulation seem to say 70ma.